pkg/promql/bench_test.go

```diff
@@ -26,7 +27,15 @@ import (
 	"github.com/prometheus/prometheus/storage"
 )
 
+func ignore() bool {
```

Comment:
Why do we ignore some benchmarks?

Reply:
We do not touch the files in promql packages, hence running benchmarks in this package is trivial.

Reply:
Is it useful to keep them if we are going to exclude them?

Also, could we change `PROMSCALE_BENCHMARK` for something more meaninful? Maybe `SKIP_PROMQL_BENCHMARKS` or use a build constraint to exclude the benchmark tests from this package.